subsys/bluetooth/audio/mcc.c

```diff
@@ -20,8 +20,8 @@
 #include <bluetooth/gatt.h>
 #include <bluetooth/audio/mcc.h>
 
-#include "otc.h"
-#include "otc_internal.h"
+#include <bluetooth/services/ots_client.h>
+#include "../services/ots/ots_client_internal.h"
```

Comment:
What does MCC need from the internal header file?

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/42445
(There is an issue to fix that one as well.)